.sync.yml

```
@@ -1,8 +1,5 @@
-# the mongodb tests currently do not work on MongoDB 3.x or newer
 ---
-.travis.yml:
-  secure: "TcZjuTP5ooDXZNRnG1M+wqmT/Lqcg8CwZfLljGlDSjDEN2siUhflHf9A81q5XL17oN4BkQtPmLvu7d/7xijUOiqDWXG9PC/Xr3QsTTmjtEo5l2Ne7jO+HrXO0Ron6LjEy47PeUWVQs7JcLRQL1SpxlNkt/Xe7oSoL10PmP2GYJc="
-  docker_sets:
-    - set: centos7-64
 appveyor.yml:
   delete: true
+.github/workflows/ci.yml:
+  pidfile_workaround: CentOS
```

Please add a new line as well
```suggestion
  pidfile_workaround: CentOS,Ubuntu
```